### ISSUE TITLE
my omniwallet is wrong

### ISSUE THREAD

**Issue by ForestCoin** [OmniLayer/omniwallet#1315]
assertion failed!
program：D\omin\omni core\omnicore-qt.exe
file:main.cpp,Line1671
Exression:hashPrevBlock == view.GetBestBlock()

**Comment by dexX7** [comment]
Did you use Bitcoin Core before, and if so, which version?

**Comment by ForestCoin** [comment]
No, I don't use bitecoin core.   I use omnicore-0.0.11.1-rel-win64-setup on my server!

**Comment by ForestCoin** [comment]
My wallet has been unable to synchronize on the server. Always prompt
assertion failed!
program：D\omin\omni core\omnicore-qt.exe
file:main.cpp,Line1671
Exression:hashPrevBlock == view.GetBestBlock()

**Comment by dexX7** [comment]
I asked, because with Bitcoin Core 0.12+ the database is obfuscated, and not necessarily compatible with Omni Core.

The error you posted indicates your block database got corrupted. Reindexing should resolve the issue.

Please open a command line (press Windows key + R, then enter cmd, hit enter) and type in the following command:

```
"D:\omin\omni core\omnicore-qt.exe" -reindex
```

This command will start Omni Core and reindex the database. This process can take a while unfortunally.

**Comment by ForestCoin** [comment]
Thank you very much

**Comment by ForestCoin** [comment]
I  want to get omni and synereo to China's echange, the virtual currency trading platform. omni use wallet like bitecoin api ?  but synereo Jjust on the Omni token don't have wallet,how to do that?

**Comment by dexX7** [comment]
Omni Core has the same API as Bitcoin Core, but with extensions:

https://github.com/OmniLayer/omnicore/blob/v0.0.11.1/src/omnicore/doc/rpc-api.md

**Comment by ForestCoin** [comment]
I use this 【"D:\omin\omni core\omnicore-qt.exe" -reindex】 but My wallet has been unable to synchronize on the server. the same problem! Always prompt
assertion failed!
program：D\omin\omni core\omnicore-qt.exe
file:main.cpp,Line1671
Exression:hashPrevBlock == view.GetBestBlock()

**Comment by ForestCoin** [comment]
new problem
Assertion failed!  
Program:D\omni core\omnicore-qt.exe  
File:main.cpp,Line1671    
Expression:hashPrevBlock == view.GetBestBlock()

**Comment by dexX7** [comment]
Please open an explorer window and navigate to `%APPDATA%/bitcoin`. Does this folder exist?

If you don't have a wallet with imported private keys or alike, which you may need to backup, please delete the whole `%APPDATA%/bitcoin` folder and start Omni Core again.

**Comment by zathras-crypto** [comment]
Hi @ForestCoin 

Sorry to hear you are having difficulty.

Could you please let us know how much RAM you have on this server?  Is it virtual?  Are you sharing it with any other altcoin or bitcoin node software?

Please try @dexX7's suggestions, and if you still get the same issue, please post the tail (last 20 lines) of your ```debug.log``` from your ```%APPDATA%\bitcoin``` directory.

Thanks

**Comment by ForestCoin** [comment]
Thank you, My wallet is in sync. but,turn off, then start to have a long time.what's reason
